tests/unit/TickingMockClockTest.php

```diff
@@ -0,0 +1,120 @@
+        $this->assertNotSame($sample1->format($timeFormat), $sample3->format($timeFormat));
+        $this->assertNotSame($sample2->format($timeFormat), $sample3->format($timeFormat));
+
+        $this->assertGreaterThan($sample1, $sample2);
+        $this->assertGreaterThan($sample1, $sample3);
+        $this->assertGreaterThan($sample2, $sample3);
+
+        TickingTimeAssertions::assertDateTimeLessThanOneSecondAfter($now, $sample1);
+        TickingTimeAssertions::assertDateTimeLessThanOneSecondAfter($now, $sample2);
+        TickingTimeAssertions::assertDateTimeLessThanOneSecondAfter($now, $sample3);
+
+        // Mutable variant.
+
+        $sample1 = $clock->nowMutable();
+        $sample2 = $clock->nowMutable();
+        $sample3 = $clock->nowMutable();
+
+        $this->assertNotSame($sample1->format($timeFormat), $sample2->format($timeFormat));
+        $this->assertNotSame($sample1->format($timeFormat), $sample3->format($timeFormat));
+        $this->assertNotSame($sample2->format($timeFormat), $sample3->format($timeFormat));
+
+        $this->assertGreaterThan($sample1, $sample2);
+        $this->assertGreaterThan($sample1, $sample3);
+        $this->assertGreaterThan($sample2, $sample3);
+
+        TickingTimeAssertions::assertDateTimeLessThanOneSecondAfter($now, $sample1);
+        TickingTimeAssertions::assertDateTimeLessThanOneSecondAfter($now, $sample2);
+        TickingTimeAssertions::assertDateTimeLessThanOneSecondAfter($now, $sample3);
+    }
+
+    /**
+     * @test
+     */
+     public function it_changes_the_time(): void
+     {
+         $timeString = '2021-05-05T14:11:49.128311';
+         $timeFormat = 'Y-m-d\TH:i:s.u';
+         $now = \DateTimeImmutable::createFromFormat($timeFormat, $timeString, new \DateTimeZone('UTC'));
+         \assert($now instanceof \DateTimeImmutable);
+         $clock = TickingMockClock::tickingFromCurrentTime($now);
+
+         $updatedTime = $now->modify('+30 minutes');
+
+         $clock->changeTimeTo($updatedTime);
+
+         TickingTimeAssertions::assertDateTimeLessThanOneSecondAfter($updatedTime, $clock->now());
+         TickingTimeAssertions::assertDateTimeLessThanOneSecondAfter($updatedTime, $clock->nowMutable());
+     }
+
+    /**
+     * @test
+     */
+    public function it_returns_a_date_object(): void
+    {
+        $timeString = '2018-04-07T16:51:29.083869';
+        $timeFormat = 'Y-m-d\TH:i:s.u';
+        $now = \DateTimeImmutable::createFromFormat($timeFormat, $timeString, new \DateTimeZone('UTC'));
+        \assert($now instanceof \DateTimeImmutable);
+        $clock = TickingMockClock::tickingFromCurrentTime($now);
+
+        $date = $clock->today();
+
+        $this->assertSame(2018, $date->year());
+        $this->assertSame(4, $date->month());
+        $this->assertSame(7, $date->day());
+    }
+
+    /**
+     * @test
+     */
+    public function it_returns_a_date_object_of_the_following_day_if_ticking_from_far_enough_back_in_the_past(): void
```

Comment:
I spent about 15 minutes to wrap my head around what is actually going on here and how it could be useful. I got there, but I wonder if a test name like 

```suggestion
    public function it_advances_the_data_once_it_has_been_ticking_for_24_hours(): void
```

May be clearer? It may also just be me that struggled with it and it's fine as-is.